### PR TITLE
Aidan main menu

### PR DESCRIPTION
**Pull request by Aidan-AIE**
- Fully implemented new main menu screen

### CODE REVIEW

File: Assets/Scripts/MainMenu/SliderSelect.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
```
Comment (SkeleKing042) [inline Assets/Scripts/MainMenu/SliderSelect.cs:3]: VISUAL SCRIPTING SPOTTED

File: Assets/Scripts/MainMenu/SoundManager.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
```
Comment (SkeleKing042) [inline Assets/Scripts/MainMenu/SoundManager.cs:3]: VISUAL SCRIPTING SPOTTED

**Review by SkeleKing042 — CHANGES_REQUESTED** [review]
thats 121 files

File: Assets/Scripts/MainMenu/SliderSelect.cs (hunk 1 above)
Comment (Aidan-AIE) [reply Assets/Scripts/MainMenu/SliderSelect.cs]: Test logo 2

**Review by SkeleKing042 — APPROVED** [review]
👍

**Pull request merged**